### PR TITLE
frontend: make locize-push; make locize-pull

### PR DESCRIPTION
**Pull request by benma**

### CODE REVIEW

File: frontends/web/src/locales/he/app.json [hunk 1]
```diff
@@ -483,7 +483,7 @@
     "accountSummaryAmount": {
       "link": {
         "text": "www.coingecko.com",
-        "url": "https://www.coingecko.com"
+        "url": "/https://www.coingecko.com"
```
[the remark this answers is not in the document]
Comment (thisconnect) [reply frontends/web/src/locales/he/app.json:486]: right to left 🤔

[the remark this answers is not in the document]
Comment (thisconnect) [reply frontends/web/src/locales/he/app.json:486]: @benma should be fixed, but I am not 100%

[the remark this answers is not in the document]
Comment (benma) [reply frontends/web/src/locales/he/app.json:486]: I fixed it. We should also maybe not put URLS into locize, makes no sense to translate these.

[the remark this answers is not in the document]
Comment (benma) [reply frontends/web/src/locales/he/app.json]: pulled again, PTAL

**Review by thisconnect — APPROVED** [review]
LGTM

**Pull request merged**